docs/src/main/paradox/index.md

```diff
@@ -7,7 +7,7 @@ part of this project as well as links to components that are maintained by other
 If you'd like to know what integrations with Alpakka look like, have a look at our 
 [self-contained examples](examples/index.html) section.
 
-There are a few blog posts and presentations about Alpakka out there, we've [collected some](other-docs.md).
+There are a few blog posts and presentations about Alpakka out there, we've [collected some](other-docs.html).
```

Comment:
That's weird, that  should work out of the box afair, paradox translates links to md-files in the docs to html links in the rendered output.

Reply:
@johanandren I'd expect that, too. But in line 8 there is already "examples/index.html".